docs/sphinx_build_multiversion.py

```diff
@@ -50,14 +50,12 @@ def build_docs_version(version):
 
 def main():
     """main function that is executed when the script is called."""
-    with open("docs/source/_static/switcher.json", "r") as f:  # pylint: disable=W1514
+    with open("docs/source/_static/switcher.json") as f:
         version_switcher = json.load(f)
 
     # get currently active branch
     command = "git rev-parse --abbrev-ref HEAD".split()
-    initial_branch = (
-        run(command, capture_output=True, check=True).stdout.strip().decode("utf-8")
-    )
+    initial_branch = run(command, capture_output=True, check=True).stdout.strip().decode("utf-8")
```

Comment:
Did you change the max length of the lines? It looks like this was changed from 88 to something larger

Reply:
Yeah I changed it to 100, I guess it's kinda arbitrary but I think most people have big enough screens to fit 100 characters, what do you think?

Reply:
On my laptop, when I review the code its a bit too long :smile:
Maybe also to keep it a bit in sync with the rest of the packages, I would opt for 88 again. But not a really strong opinion here. Your call

Reply:
ah ok! I have reverted it to 88 :)